packages/react-vapor/src/components/actions/Action.tsx

```diff
@@ -51,22 +51,25 @@ export class Action extends React.Component<IActionProps, any> {
     };
 
     render() {
-        const actionIcon: JSX.Element = this.props.action.icon ? (
-            <Svg svgName={this.props.action.icon} className="action-icon" svgClass="icon fill-medium-blue" />
+        const {icon, id, name, tooltipPlacement, tooltip} = this.props.action;
+        const actionIcon: JSX.Element = icon ? (
+            <Svg svgName={icon} className="action-icon" svgClass="icon fill-medium-blue" />
         ) : (
             <Svg svgName="more" className="action-icon action-icon-more" svgClass="icon icon-medium fill-medium-blue" />
         );
         const inside: string | JSX.Element = this.props.simple ? (
-            this.props.action.name
+            name
         ) : (
             <span className="inline-flex flex-center">
                 {actionIcon}
-                <span className="action-label">{this.props.action.name}</span>
+                <span className="action-label" data-trigger={id || name}>
+                    {name}
+                </span>
             </span>
         );
-        const tooltipPlacement: string = this.props.action.tooltipPlacement || 'right';
-        const wholeAction: JSX.Element = this.props.action.tooltip ? (
-            <Tooltip title={this.props.action.tooltip} placement={tooltipPlacement}>
+        const placement: string = tooltipPlacement || 'right';

```

Comment:
```suggestion
        const placement: string = tooltipPlacement || TooltipPlacement.Right;
```
You will need to import `TooltipPlacement` however.